### ISSUE TITLE
JetEngine Ability to create user roles

### ISSUE THREAD

**Issue by moxet** [CrocoBlock/suggestions#6670]
We usually install 3rd party plugin just to create user roles, it would be nice if the next version of JetEngine include user role creation as well as display the role in dynamic field.

**Comment by Picrold** [comment]
+1111

**Comment by sekhsoyebali** [comment]
+1

**Comment by 79ho3ein** [comment]
+1

**Comment by LevelUpGit** [comment]
I don't think this is needed, as role creation, and permissions management are a lot of work to do. [AAM] (https://wordpress.org/plugins/advanced-access-manager/) is perfect for this stuff, and much more, and it's working perfectly with JetEngine and it's high quality plugin. While I agree sometime is stupid that we have to install separate plugins, but not for this one, best is to put effort into polishing current products.

**Comment by moxet** [comment]
@LevelUpGit If you have used jetengine you must know that the rest of permissions and visibility is controlled by JetEngine itself. We need to create roles only. 

I know it can be done with few lines of code but as a frequent user of jetengine it will save a lot of time for us.

**Comment by NeoAcla** [comment]
@MjHead @Crocoblock , it's been more than 2 years old of this request and some previous request is more than this.
Are you guys still not considering this? A JetRole should be created specially with dynamic visibility is already there.
A lot of users are requesting this for a long time.

Yes we are aware and we do use a third party plugin, but having on crocoblock plugins list is different.
